NEWS.md

```diff
@@ -1,3 +1,28 @@
+# qCreate 0.1.1
+
+## Package
+
+* Closed #40 by adding the helper function `correct_words()` into `standardise_titles()`
+* Added `code_term()` which extracts the date term from the treaty title
+* Added `code_grounds()` which indicates the type of termination clause the treaty has
+* Added tests for `code_term()` and `code_grounds()` functions
```

Comment:
Under `Correction`? Are you citing all the issues closed in the NEWS/PR description?

Reply:
Yes, all the issues closed/awaiting deployment for this cycle are included in the PR. I have, however, re-shuffled the bullet points around to maintain the consistency with which functions were coded